Doc/library/faulthandler.rst

```diff
@@ -66,10 +66,22 @@ Dumping the traceback
       Added support for passing file descriptor to this function.
 
 
+Dumping the C stack
+-------------------
+
+.. versionadded:: next
+
+.. function:: dump_c_stack(file=sys.stderr)
+
+   Dump the C stack trace of the current thread into *file*.
+
+   If the system doesn't support the C-level :manpage:`backtrace` or :manpage:`backtrace_symbols` functions,
+   then an error message is displayed instead of the C stack.
```

Comment:
```suggestion
   If the system does not support the C-level :manpage:`backtrace(3)`
   or :manpage:`backtrace_symbols(3)` functions, then an error message
   is displayed instead of the C stack.
```